libs/components/src/lib/date-picker/date-picker.spec.ts

```diff
@@ -253,6 +327,42 @@ describe('vwc-date-picker', () => {
 
 			expect(textField.errorText).toBe('');
 		});
+
+		it('should move focus to the tabbable element when pressing tab in the text-field', async () => {
+			await openPopup();
+
+			pressKey('Tab');
+
+			expect(element.shadowRoot!.activeElement).toBe(
+				getDateButton('2023-08-10')
```

Comment:
Would that be more understandable?
```suggestion
                        const firstTabbableElement = getDateButton('2023-08-10');
			expect(element.shadowRoot!.activeElement).toBe(firstTabbableElement);
```

Reply:
Yes, done